examples/kcp/go.mod

```diff
@@ -0,0 +1,123 @@
+module github.com/kcp-dev/controller-runtime/examples/kcp
+
+go 1.22.2
+
+replace sigs.k8s.io/controller-runtime => ../../
```

Comment:
add a comment that this must like "like this" for the example to run standalone.

Reply:
added